bin/mkreleaselog

```diff
@@ -1,40 +1,75 @@
 #!/bin/zsh
-#set -x
 set -euo pipefail
 export GO111MODULE=on
 export GOPATH="$(go env GOPATH)"
 
-alias jq="jq --unbuffered"
-
-AUTHORS=(
+# List of PCRE regular expressions to match "included" modules.
+INCLUDE_MODULES=(
     # orgs
-    ipfs
-    ipld
-    libp2p
-    multiformats
-    filecoin-project
-    ipfs-shipyard
-
-    # Authors of personal repos used by go-ipfs that should be mentioned in the
+    "^github.com/ipfs/"
+    "^github.com/ipld/"
+    "^github.com/libp2p/"
+    "^github.com/multiformats/"
+    "^github.com/filecoin-project/"
+    "^github.com/ipfs-shipyard/"
+
+    # Authors of personal modules used by go-ipfs that should be mentioned in the
     # release notes.
-    whyrusleeping
-    Kubuxu
-    jbenet
-    Stebalien
-    marten-seemann
-    hsanjuan
-    lucas-clemente
-    warpfork
+    "^github.com/whyrusleeping/"
+    "^github.com/Kubuxu/"
+    "^github.com/jbenet/"
+    "^github.com/Stebalien/"
+    "^github.com/marten-seemann/"
+    "^github.com/hsanjuan/"
+    "^github.com/lucas-clemente/"
+    "^github.com/warpfork/"
+)
+
+# List of PCRE regular expressions to match "excluded" modules. Applied after includes.
+EXCLUDE_MODULES=(
+    "^github.com/marten-seemann/qtls"
```

Comment:
This needs to be a regex. For example, for Go 1.16 the repo is https://github.com/marten-seemann/qtls-go1-16.

Reply:
It's a regex. It'll match anything that starts with `github.com/marten-seemann/qtls`.

Reply:
🤦‍♂️, sorry.